src/m365/tenant/commands/people/people-profilecardproperty-get.ts

```diff
@@ -56,8 +56,11 @@ class TenantPeopleProfileCardPropertyGetCommand extends GraphCommand {
         await logger.logToStderr(`Retrieving information about profile card property '${args.options.name}'...`);
       }
 
+      // Get the right casing for the profile card property name
+      const profileCardProperty = profileCardPropertyNames.find(p => p.toLowerCase() === args.options.name.toLowerCase());
+
       const requestOptions: CliRequestOptions = {
-        url: `${this.resource}/v1.0/admin/people/profileCardProperties/${args.options.name}`,
+        url: `${this.resource}/v1.0/admin/people/profileCardProperties/${profileCardProperty}`,
```

Comment:
Should we really enforce the casing for the `get` command? Folks could always add properties straight to the graph API with their own casing. If they then want to get the property through the CLI they'll get an empty result while it exists with different casing.

The same goes for the `remove` and `set` commands.

Reply:
Yes, but on the other hand if they specify for example `FAX` in the add command and use the same value for the remove or get command, they will receive an error that the property doesn't exist, which is weird.

In my opinion we do it everywhere or nowhere at all.

Reply:
I agree to keep consistent behavior across the commands.